sdks/besu/interoperation-node-sdk/build/AssetManager.js

```diff
@@ -0,0 +1,205 @@
+"use strict";
```

Comment:
Don't commit the build directory, add it to `.gitignore` here. This can be generate by running `make build` right?